maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -115,6 +120,14 @@ public class EnforceMojo extends AbstractMojo {
     @Parameter(required = false, property = "rules")
     private String[] commandLineRules;
```

Comment:
The property name `rules` seems wrong. I would expect the `enforcer.` prefix here. 

We can't really rename the property at this point since that would be a breaking change, but maybe we could deprecate it and introduce new one named `enforcer.rules` which would be the replacement?

Reply:
deprecated will be good idea - separate issue and PR